### ISSUE TITLE
CARD component

### ISSUE THREAD

**Issue by ninabreznik** [ethereum-play/smartcontract.codes#8]
## @TODO

- [ ] make the card responsive also on very small width (when only one card is left and you make the width smaller, the card should still be responsive) (@fionataeyang)

## @LATER

- [ ] implement custom scrollbar to see the code when on hover (make custom nicely styled scroll bars) - only up & down, not left & right (??)

- [ ] implement code mirror (to get syntax highlighting, line breaks/responsivenes of the code) - therefore no need for left & right scrolling (??)

- [ ] adapt highlighting to our theme (use code mirror) (??)

**Comment by fionataeyang** [comment]
@ninabreznik The card was already responsive as well. I found why it's caused not perfectly responsive very well. Coz searbar was changed with a fixed with. I add a style of the small screen's width. It's worked fine now.

The words in syntax highlight are break now, just add style to make it fit.
I don't really think we need to add the scrollbar into the card design. 
It makes the users hard to read on it from a small card, I think we should add a extend button to open it larger to like a big area (like article).
